client/src/components/LinkDisplay/style.css

```diff
@@ -0,0 +1,36 @@
+.container {
```

Comment:
let scope it with an id probably. 
Otherwise, generic class names will cause conflicts later and it'll be extremely hard to debug.